.github/workflows/release-types-package.yml

```diff
@@ -87,7 +87,7 @@ jobs:
         run: npm run depcheck
 
   check-version:
-    name: 'Check version'
+    name: "Check version"
```

Comment:
there's some formatting in the workflow that looks good, removing white space and fixing indents. I'm not sure about the change from single `'` to double `"`. The convention seems to be single `'` except for in `run:` commands

Reply:
I've left the cargo.toml files as we normally have them

Reply:
It only seems to affect Node.